advanced_topics.qmd

```diff
@@ -375,15 +381,17 @@ plot <- ggplot(model_data) +
 ::: {.content-visible when-format="html"}
 ```{r}
 #| label: plot_fit_data_plot
+#| warning: false
 #| echo: false
 plot
 ```
-::: 
+:::
 
 ::: {.content-visible when-format="pdf"}
 ```{r}
 #| label: plot_fit_data_plot-pdf
 #| echo: false
+#| warning: false
```

Comment:
Thanks for adding this. I realized when I looked at the preview I had it but then when I split web vs pdf I copy pastaed it out of existence and hadn't remembered to put it back